invenio/legacy/dbquery_postgresql.py

```diff
@@ -225,30 +225,9 @@ def get_table_update_time(tablename, run_on_slave=False):
        wildcard `%' in which case we return the maximum update time
        value.
     """
-    # Note: in order to work with all of MySQL 4.0, 4.1, 5.0, this
-    # function uses SHOW TABLE STATUS technique with a dirty column
-    # position lookup to return the correct value.  (Making use of
-    # Index_Length column that is either of type long (when there are
-    # some indexes defined) or of type None (when there are no indexes
-    # defined, e.g. table is empty).  When we shall use solely
-    # MySQL-5.0, we can employ a much cleaner technique of using
-    # SELECT UPDATE_TIME FROM INFORMATION_SCHEMA.TABLES WHERE
-    # table_name='collection'.
-    res = run_sql("SHOW TABLE STATUS LIKE %s", (tablename,),
-                  run_on_slave=run_on_slave)
-    update_times = [] # store all update times
-    for row in res:
-        if type(row[10]) is long or \
-           row[10] is None:
-            # MySQL-4.1 and 5.0 have creation_time in 11th position,
-            # so return next column:
-            update_times.append(str(row[12]))
-        else:
-            # MySQL-4.0 has creation_time in 10th position, which is
-            # of type datetime.datetime or str (depending on the
-            # version of MySQLdb), so return next column:
-            update_times.append(str(row[11]))
-    return max(update_times)
+    # FIXME how can I implement it with PostgreSQL???
```

Comment:
@jirikuncar do you have some idea on how can I make an equivalent function for postgresql? T_T